Components/Card/card.css

```diff
@@ -0,0 +1,109 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap');
+
+@import url("../Buttons/Buttons.css");
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Playfair Display', serif;
+}
+
```

Comment:
It would be better to make a `styles.css/main.css` in the root and put these properties there and import that main stylesheet into your HTML file as these are recurring properties.

Also, there are some commented out properties in the CSS files, please remove them.

Reply:
I will refactor that in the end. This was said to me for other components. I have removed the commented codes from css.